File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/BackupOrderPutRequest.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Parameters to submit to validate your backup order
 */
class BackupOrderPutRequest
{
    #[DTA\Data(field: "validateOnly", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "bool"])]
    public bool|null $validate_only = null;

    #[DTA\Data(field: "serviceType", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $service_type = null;

    #[DTA\Data(field: "coupon", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $coupon = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/BackupOrderPutRequest.scala
package io.swagger.server.model


/**
 * = Root Type for BackupOrderPutRequest =
 *
 * Parameters to submit to validate your backup order
 *
 * @param validateOnly 
 * @param serviceType 
 * @param coupon 
 */
case class BackupOrderPutRequest (
  validateOnly: Option[Boolean],
  serviceType: Option[Int],
  coupon: Option[String]
)

